src/import/saga.ts

```diff
@@ -241,7 +241,7 @@ function* attemptBackupPhraseCorrection(mnemonic: string) {
 function* walletHasBalance(address: string) {
   Logger.debug(TAG + '@walletHasBalance', 'Checking account balance')
   const tokenBalances: FetchedTokenBalance[] = yield* call(fetchTokenBalancesForAddress, address)
-  return tokenBalances.length > 0
+  return tokenBalances.filter((token) => token.balance !== '0').length > 0
```

Comment:
The backend returns some tokens even with zero balance.

Reply:
Good catch! Maybe this could have been done separately to give higher visibility into this fix?

Reply:
Ok, I'll put this in another PR :)